### DISCUSSION TITLE
VS Code Speech User Interviews

### DISCUSSION THREAD

**Discussion by isidorn** [microsoft/vscode-discussions#1144] [Announcements]
We are actively developing our [speech extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-speech). We want to know how to improve it and and we are looking for your guidance to help us understand how you use it today and what you would like to be able to do with it in the future.
 
You can help us by joining us in a research session with the team. Each session will last for one hour and during the session you'll discuss how you work today and how the speech extension impacts the way you work. You'll then work on a series of small tasks to explore the future of the speech experience in VS Code. As a current user of the speech extension, your input and insights will be extremely valuable and will help to improve the experience for you and for everybody else that uses the extension.
 
If you are a user of the Speech Extension and would like to join us on one of our research sessions, please comment here or send an email to inikolic@microsoft.com.
If you use the Speech Extension with a non-English language that is super interesting for us, as we would like to learn how the experience fares with non-English languages.
 
If you don't currently use the Speech Extension you can learn more about it here: https://code.visualstudio.com/docs/editor/voice
 
Thank you!

**Comment by stevencl** [comment]
If you would like to discuss how we can improve the speech extension, we are using UserInterviews.com to manage these sessions. Please sign up at this link: https://www.userinterviews.com/projects/RBR4YN8_mA/apply. If you don't already have a UserInterviews account you'll be asked to create one, then you can choose to sign up for our project. If you end up doing a session with us we'll show our appreciation for your time by sending you a gift card of your choice.

If you know of anybody else who might be interested in sharing their experience with the speech extension, please pass on the link and encourage them to sign up.

Thanks!
